JsonSchema/SchemaKeywordRegistry.cs

```diff
@@ -173,6 +172,13 @@ public static void Unregister<T>()
 			: null;
 	}
 
+	internal static JsonTypeInfo GetTypeInfo(Type ruleType)
+	{
+		return _keywordTypeInfoResolvers.TryGetValue(ruleType, out var context)
+			? context.GetTypeInfo(ruleType)!
+			: JsonSchemaSerializerContext.Default.GetTypeInfo(typeof(UnrecognizedKeyword))!;
+	}
```

Comment:
Why is this ignoring ruleType here and passing UnrecognizedKeyword explicitly?

Reply:
If the keyword isn't found, it's treated as an unrecognized keyword.  This captures the data for the keyword.  It doesn't play into validation and just generates and annotation.